src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Square - side:" + side + "and area is: " + calculateArea());
```

Comment:
There is a missing space after 'side:' and before 'and area is: ' in the draw method's output, which could affect readability. Consider adding a space for better formatting.